docs/kyma/docs/017-details-local-installation.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Local installation
+type: Details
+---
+
+For better understanding of complex installation process, this guide describes it step by step.
+
+## Start local installation
+
+To fire local installation run following command:
+```
+./installation/cmd/run.sh
+```
+
+This script sets up default parameters, starts minikube, builds kyma-installer, generates local configuration, creates `installer` Custom Resource and sets up installer. Subsequent sections provide a detailed description of each step.
+
+### Installation parameters
+
+You can execute the `installation/cmd/run.sh` script with the following parameters:
+
+- `--skip-minikube-start` - it skips execution of the `installation/scripts/minikube.sh` script. See the "Start Minikube" section for more details.
+- `--vm-driver` -  either `virtualbox` or `hiperkit` depending on your operating system
+
+## Start minikube
+
+> **NOTE:** To work with Kyma, use only the provided scripts and commands. Kyma does not work on a basic Minikube cluster that you can start using the `minikube start` command or stop with the `minikube stop` command. If you don't need Kyma on Minikube anymore, remove the cluster with the `minikube delete` command.
+
+Purpose of `installation/scripts/minikube.sh` is to configure and start minikube. The script also checks if your development environment is cofigured to handle kyma installation. This includes checking minikube and kubectl versions. In case minikube is already initialized you will be prompted to agree to remove previous minikube cluster. Script exits if you don't want to restart your cluster.
+
+>**NOTE:** Minikube is configured to disable default nginx ingress controller.
+
+>**NOTE:** For the complete list of parameters passed to `minikube start` command, refer to the `installation/scripts/minikube.sh` script.
+
+Once minikube is up and running, the script adds develop domains to /ets/hosts.
+
+## Build kyma-installer
+
+Kyma installs locally using a proprietary installer based on a [Kubernetes operator](https://coreos.com/operators/).
```

Comment:
Reword this paragraph - it is not clear.

Reply:
i removed this sentence completely